### PR TITLE
add conway config for mainnet/preprod/preview

### PR DESCRIPTION
**Pull request by disassembler**

### CODE REVIEW

**Comment by Ryun1** [comment]
Is there a need to add the Conway genesis to [preprod-config-bp.nix](https://github.com/input-output-hk/iohk-nix/blob/master/cardano-lib/preprod-config-bp.nix#L11) / [preview-config-bp.nix](https://github.com/input-output-hk/iohk-nix/blob/master/cardano-lib/preview-config-bp.nix#L11)?

**Comment by disassembler** [comment]
yes, I'll add them as soon as we're confidnet on mainnet values. It's a pain to regenerate hashes every single change :laughing:

**Comment by Ryun1** [comment]
> yes, I'll add them as soon as we're confidnet on mainnet values. It's a pain to regenerate hashes every single change 😆

Understood
I was just looking for anything I could ask

**Review by kevinhammond — APPROVED** [review]
I have checked the parameter settings against the Parameter Committee recommended values, the Plutus v3 cost model against the SanchoNet settings, validated the constitution hash, and confirmed the hash of the guardrails script against the provided one.

File: cardano-lib/mainnet/conway-genesis.json [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+      "anchor": {
+          "dataHash": "ca41a91f399259bcefe57f9858e91f6d00e1a38d6d9c63d4052914ea7bd70cb2",
+          "url": "ipfs://QmVyzwKe8NmswkVVn2LKCorFEctKCxeBApA6d89HKWd3qH"
+      },
+      "script": "fa24fb305126805cf2164c161d852a0e7330cf988f1fe558cf7d4a64"
+  },
+  "committee": {
+    "members": {
+        "keyHash-df0e83bde65416dade5b1f97e7f115cc1ff999550ad968850783fe50": 580,
+        "keyHash-b6012034ba0a7e4afbbf2c7a1432f8824aee5299a48e38e41a952686": 580,
```
Comment (Ryun1) [inline cardano-lib/mainnet/conway-genesis.json:291]:
```suggestion
        "scriptHash-b6012034ba0a7e4afbbf2c7a1432f8824aee5299a48e38e41a952686": 580,
```

does it matter if these are correctly labelled as keyHashes or scriptHashes?

**Review by Ryun1 — APPROVED** [review]
I can confirm Intersect's script-based credentials are present within the Mainnet and PreProd genesis files ✅

**Comment by disassembler** [comment]
I'll merge this first thing tomorrow around 1300 UTC.

**Comment by thenic95** [comment]
LGTM!

The `conway-genesis.json` governance parameter values align with the recommendations of the Parameter Committee.

**Pull request merged**